src/PubSub/PubSubClient.php

```diff
@@ -357,6 +357,8 @@ function (array $subscription) {
      * @param Subscription $subscription The subscription to take a snapshot of.
      * @param array $options [optional] Configuration options.
      * @return Snapshot
+     * @experimental
+     * @whitelisted
```

Comment:
The link isn't working for me. I think you're speaking about `PubSubClient::snapshot()`? There's no whitelist description because the method doesn't perform a service call. It's not super useful to instantiate a Snapshot instance without the feature, but it doesn't break anything.